tests/Testing/StreamxResponse.php

```diff
@@ -29,11 +29,32 @@ public static function successResultWithFailure(string $errorCode, string $error
         $failureResponse = new FailureResponse($errorCode, $errorMessage);
         $messageStatus = MessageStatus::ofFailure($failureResponse);
         $json = json_encode($messageStatus);
-        return new Response($json, [], 202);
+        return self::create202Response($json);
+    }
+
+    /**
+     * @param $messageResponses array of SuccessResult and/or FailureResponse $messageResponses
+     */
+    public static function responseForMultipleMessages(array $messageResponses): Response
+    {
+        $jsons = [];
+        foreach ($messageResponses as $messageResponse) {
+            if ($messageResponse instanceof SuccessResult) {
+                $jsons[] = json_encode(MessageStatus::ofSuccess($messageResponse));
+            } else if ($messageResponse instanceof FailureResponse) {
+                $jsons[] = json_encode(MessageStatus::ofFailure($messageResponse));
+            }
+        }
+        return self::create202Response(implode("\n", $jsons));
     }
```

Comment:
Note: we mock StreamX response here, but this PR contains also an integration test executed against a running StreamX instance (for publishing and unpublishing multimessage json)